src/mgr/BaseMgrModule.cc

```diff
@@ -570,6 +570,12 @@ ceph_get_version(BaseMgrModule *self, PyObject *args)
   return PyString_FromString(pretty_version_to_str().c_str());
 }
 
+static PyObject *
+ceph_get_release_name(BaseMgrModule *self, PyObject *args)
```

Comment:
so we can drop `args` ?

Reply:
> so we can drop `args` ?

I'm not sure, all functions that are using ``METH_NOARGS`` still have the ``PyObject *args`` parameter.

Reply:
@votdev sorry, i was wrong. and you are right. see https://github.com/python/cpython/blob/32d96a2b5bc3136d45a66adbdb45fac351b520ce/Modules/_sha3/clinic/sha3module.c.h#L17-L21

we can use `Py_UNUSED` for silencing compiler warnings. but it was introduced in python3.4.

probably we can use something like
```c++
static PyObject *
ceph_get_release_name(BaseMgrModule *self, PyObject *)
```

Reply:
Did a cleanup for other functions, too.

Reply:
> @votdev sorry, i was wrong. and you are right. see [python/cpython:Modules/_sha3/clinic/sha3module.c.h@`32d96a2`#L17-L21](https://github.com/python/cpython/blob/32d96a2b5bc3136d45a66adbdb45fac351b520ce/Modules/_sha3/clinic/sha3module.c.h#L17-L21)
> 
> we can use `Py_UNUSED` for silencing compiler warnings. but it was introduced in python3.4.
> 
> probably we can use something like
> 
> ```c++
> static PyObject *
> ceph_get_release_name(BaseMgrModule *self, PyObject *)
> ```

Hmm, after cleanup i do not see compiler warnings and the build succeeds. How shall we proceed?

Reply:
OK, https://docs.python.org/3/c-api/structures.html 

> Hmm, after cleanup i do not see compiler warnings and the build succeeds. How shall we proceed?

https://docs.python.org/3/c-api/structures.html#c.PyCFunction is answering my question. So we should use your approach.

Reply:
@tchaikov What about the solution to define to Py_UNUSED macro for Python2? See the second commit of this PR.

Reply:
@votdev i just don't like the compiler warnings. `Py_UNUSED` macro just makes this explicit. i still think
```c++
static PyObject *
ceph_get_release_name(BaseMgrModule *self, PyObject *)
```
is a simpler approach to get away from this.

but this is not critical.